config/src/projects/nahmii.ts

```diff
@@ -0,0 +1,103 @@
+import { Project } from './Project'
+
+export const nahmii: Project = {
+  name: 'Nahmii 1.0',
+  slug: 'nahmii',
+  bridges: [
+    {
+      address: '0xCc8D82f6ba952966E63001c7B320EEF2Ae729099',
+      sinceBlock: 6921581,
+      tokens: ['NII', 'HBT', 'HOT'],
```

Comment:
This configuration is problematic.

1) The HBT token actually refers to the `Habitat Token`, and you want to refer to `Hubiits` which is not present in the token list and also has the HBT symbol. Adding this token violates our code & UI assumption that each token has a unique symbol and will require some UI changes to properly handle.

2) The `HOT` token seems to be locked in the contract by mistake and does not meaningfully ($0.20) contribute to the TVL.

My suggestion is as follows: In the long term we would display NII and the correct HBT. In the short term if you want this PR merged quicker you can say:
```
// should also include Hubiits (HBT) - 0xDd6C68bb32462e01705011a4e2Ad1a60740f217F
tokens: ['NII'],
```